### DISCUSSION TITLE
Health Check for docker container

### DISCUSSION THREAD

**Discussion by alternativ** [go-shiori/shiori#964] [Feature requests]
Request is fairly simple, we need a healthcheck for the container. Currently trying to curl or wget or any other mechanism drops as they echo that the `executable file not found in $PATH: unknown`. This might have to do with the way the docker container is setup and run using an entrypoint. Further debugging would be required.

**Comment by fmartingr** [comment]
Yeah, currently the docker container uses a very minimal filesystem. The idea would be to provide different tags for the same version of the same image with more available tooling for debugging, but the minimal one probably would need curl as well for the healthcheck since that's fairly basic.
